seshdash/views.py

```diff
@@ -1388,7 +1379,7 @@ def delete_report(request, report_id):
     return redirect(reverse('manage_reports', args=[site.id]))
 
 @login_required
```

Comment:
is this method secure? We know login is needed but what if a logged in users uses a form to POST with a site id he doesn't have permission to use?